openapi-cli/src/main/java/org/ballerinalang/generators/BallerinaSchemaGenerator.java

```diff
@@ -315,7 +315,7 @@ public static OpenAPI parseOpenAPIFile(String definitionURI) throws OpenApiExcep
         if (!Files.exists(contractPath)) {
```

Comment:
Shall we refactor `parseOpenAPIFile` a bit?
- Access can be private
- `parseResult.getMessages().size() > 0` can be replaced with `!parseResult.getMessages().isEmpty()`
- `api` variable seems redundant, can immediately return `return parseResult.getOpenAPI()`